depenguinme.sh

```diff
@@ -390,6 +390,7 @@ if [ "$USE_IPV6" = "YES" ]; then
 	set -e
 fi
 
+# shellcheck disable=SC2086
```

Comment:
hi @bretton 

I think it would be better to change the code

```
kill $keyscan_pid >/dev/null 2>&1
```

to

```
kill "$keyscan_pid" >/dev/null 2>&1
```

And remove the shellcheck exclusion.